sdk/src/main/java/com/google/cloud/dataflow/sdk/options/DataflowPipelineDebugOptions.java

```diff
@@ -209,6 +209,19 @@ public Dataflow create(PipelineOptions options) {
   void setNumberOfWorkerHarnessThreads(int value);
 
   /**
+   * If {@literal true}, save a heap dump before killing a thread or process which is GC
+   * thrashing or out of memory.
+   *
+   * <p>
+   * CAUTION: Heap dumps can of comparable size to the default boot disk. Consider increasing
+   * the boot disk size before setting this flag to true.
+   */
+  @Description("If {@literal true}, save a heap dump before killing a thread or process "
```

Comment:
Ah, no it isn't  - was thrown by path (eventually this should go into a runners/dataflow dir in root).